### PR TITLE
Add Incl/Excl Decision Components

### PR DESCRIPTION
**Pull request by dhaselhan**
* Add Inclusion and Exclusion Decision Components with Copy Feature
* Add new decision component column to store applicant type

### CODE REVIEW

File: alcs-frontend/src/app/features/application/decision/decision-v2/decision-input/decision-components/decision-component/decision-component.component.ts [hunk 1]
```diff
@@ -112,6 +115,12 @@ export class DecisionComponentComponent implements OnInit {
         case APPLICATION_DECISION_COMPONENT_TYPE.SUBD:
           this.patchSubdFields();
           break;
+        case APPLICATION_DECISION_COMPONENT_TYPE.INCL:
+          this.patchInclExclFields();
+          break;
+        case APPLICATION_DECISION_COMPONENT_TYPE.EXCL:
+          this.patchInclExclFields();
+          break;
```
Comment (mhuseinov) [inline alcs-frontend/src/app/features/application/decision/decision-v2/decision-input/decision-components/decision-component/decision-component.component.ts:123]: if the component props are the same you could do 
`      case APPLICATION_DECISION_COMPONENT_TYPE.INCL:
        case APPLICATION_DECISION_COMPONENT_TYPE.EXCL:
          this.patchInclExclFields();
          break;`

File: alcs-frontend/src/app/features/application/decision/decision-v2/decision-input/decision-components/decision-component/decision-component.component.ts [hunk 2]
```diff
@@ -176,6 +185,12 @@ export class DecisionComponentComponent implements OnInit {
       case APPLICATION_DECISION_COMPONENT_TYPE.SUBD:
         dataChange = { ...dataChange, ...this.getSubdDataChange() };
         break;
+      case APPLICATION_DECISION_COMPONENT_TYPE.INCL:
+        dataChange = { ...dataChange, ...this.getInclExclDataChange() };
+        break;
+      case APPLICATION_DECISION_COMPONENT_TYPE.EXCL:
+        dataChange = { ...dataChange, ...this.getInclExclDataChange() };
+        break;
```
Comment (mhuseinov) [inline alcs-frontend/src/app/features/application/decision/decision-v2/decision-input/decision-components/decision-component/decision-component.component.ts:193]: same as above

**Review by mhuseinov — APPROVED** [review]

File: alcs-frontend/src/app/features/application/decision/decision-v2/decision-input/decision-components/decision-component/decision-component.component.ts (hunk 1 above)
Comment (dhaselhan) [reply alcs-frontend/src/app/features/application/decision/decision-v2/decision-input/decision-components/decision-component/decision-component.component.ts]: done! Good suggestion.

File: alcs-frontend/src/app/features/application/decision/decision-v2/decision-input/decision-components/decision-component/decision-component.component.ts (hunk 2 above)
Comment (dhaselhan) [reply alcs-frontend/src/app/features/application/decision/decision-v2/decision-input/decision-components/decision-component/decision-component.component.ts]: done!

**Pull request merged**